src/Todo.css

```diff
@@ -1,20 +1,38 @@
-li {
-  background-color: white;
+.todo {
+  background-color: #ffffff;
   width: 80vw;
   padding: 25px 30px;
```

Comment:
Use `inline-start/end` and `block-start/end` keywords words for `margin/padding` instead of `top/right/bottom/left`.